Mantis.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |s|
   s.social_media_url   = "http://twitter.com/guoyingtao"
   s.platform     = :ios
   s.swift_version = "5.0"
-  s.ios.deployment_target = "11.0"
```

Comment:
Same question as here: https://github.com/guoyingtao/Mantis/pull/177#discussion_r917687932

Reply:
It is because of my previous PR https://github.com/guoyingtao/Mantis/pull/169
I added a CropToolbarIconProvider also I decided to use SF Symbol for CropToolbar icons instead of drawing them by myself, which needs iOS 13.
With apple's iOS 16 coming soon, I think it might be a good timing to move to iOS 13 because it also supports Combine, SwiftUI etc.

Reply:
I know that iOS 16 is coming soon, and from a developers perspective we always want higher min versions. I personally still support up to 12+

Other than asking nicely, is there anything I can do so that you consider supporting 12+? I know the benefit of using SF Symbols is nice, but it's not even like we are able to replace all images with symbols.

Reply:
I'd very much appreciate that.